tasks/bundle-css.js

```diff
@@ -41,7 +41,7 @@ module.exports = createTask(
 ).addWatcher(
     ['src/**/*.less'],
     async () => {
-        await bundleCSS({production: false});
+        await bundleCSS({debug: false});
```

Comment:
This should be `debug: true`. Later I will make it pass the options into this function.